.github/dependabot.yml

```diff
@@ -1,16 +1,44 @@
 # Please see the documentation for all configuration options:
 # https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
 
+__: &npm_config
+  package-ecosystem: "npm"
+  schedule:
+    interval: "weekly"
+  open-pull-requests-limit: 2
```

Comment:
I wonder if github will complain about this config, otherwise I can't reuse the block

Reply:
and I have to repeat myself as in https://github.com/lens-protocol/lens-sdk/pull/185/commits/fc01a6d239b37d11000eac978c897e8e760f1aeb